.github/workflows/docker.yml

```diff
@@ -0,0 +1,68 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  id-token: write
+  packages: write
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=ref,event=branch
+            type=sha
+            type=raw,value=latest
+
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Login to GHCR
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v4
+        with:
+          platforms: linux/amd64,linux/arm64
```

Comment:
We specify two platforms here, which makes sense. How did the `unknown/unknown` combo, which I see on your fork, come into being, though?

Reply:
I found [this](https://github.com/orgs/community/discussions/45969) discussion.

let's see if https://github.com/MarcoIeni/cargo-http-registry/commit/b55bec101844db61cacf2c61265b90df4a590e36 fixes it.

Reply:
it worked!
![image](https://github.com/d-e-s-o/cargo-http-registry/assets/11428655/42085c9a-68bf-45bf-b6e5-9d3d2668c96e)